tools/props/Versions.props

```diff
@@ -18,7 +18,7 @@
     <MicrosoftIdentityClientVersion>4.7.1</MicrosoftIdentityClientVersion>
     <MicrosoftIdentityModelProtocolsOpenIdConnectVersion>5.6.0</MicrosoftIdentityModelProtocolsOpenIdConnectVersion>
     <MicrosoftIdentityModelJsonWebTokensVersion>5.6.0</MicrosoftIdentityModelJsonWebTokensVersion>
-    <SystemBuffersVersion>4.5.0</SystemBuffersVersion>
+    <SystemBuffersVersion>4.5.1</SystemBuffersVersion>
```

Comment:
I don't mind updating dependencies, they should also be updated in Nuspec file.

Reply:
Done.